docs/apache-airflow/core-concepts/executor/local.rst

```diff
@@ -47,3 +47,14 @@ Arguably, :class:`~airflow.executors.sequential_executor.SequentialExecutor` cou
 parallelism of just 1 worker, i.e. ``self.parallelism = 1``.
 This option could lead to the unification of the executor implementations, running
 locally, into just one :class:`~airflow.executors.local_executor.LocalExecutor` with multiple modes.
+
+.. note::
+
+   When using the ``@task`` decorator with multiple Schedulers configured with ``executor = LocalExecutor`` in the [core] section of airflow.cfg, each scheduler will run a LocalExecutor. This means tasks would be processed in a distributed fashion across the machines running the Schedulers. 
```

Comment:
Does this apply only when using the @task decorator? Or it is a general behaviour. If general, can we remove the leading part of the sentence which mentions about the case for @task decorator?

Reply:
My understanding is that it is general

Reply:
@pankajkoti Updated and done